src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventTableStorageRepository.cs

```diff
@@ -101,43 +109,93 @@ internal CloudTable GetDiagnosticEventsTable(DateTime? now = null)
             return _diagnosticEventsTable;
         }
 
-        private static string GetCurrentTableName(DateTime now)
+        private static string GetTableName(DateTime date)
         {
-            return $"{TableNamePrefix}{now:yyyyMM}";
+            return $"{TableNamePrefix}{date:yyyyMM}";
         }
 
         protected internal virtual async void OnFlushLogs(object state)
         {
             await FlushLogs();
         }
 
+        private async Task PurgePreviousEventVersions()
+        {
+            _logger.LogTrace("Purging previous diagnostic event versions.");
```

Comment:
Should we also include what the current Event version value is at the time of this operation? Would it add any value?

Reply:
Good idea